### DISCUSSION TITLE
Is there a way to open 2 subtiles at the same time, but not in translator mode?

### DISCUSSION THREAD

**Discussion by JustAndreww** [SubtitleEdit/subtitleedit#8317] [Q&A]
Hello!

Is there a way to open 2 subtiles at the same time, but not in translator mode? I don't want to create subtitles that will match the original ones, instead I need just to see the text from them with correct timing while editing my subtitles.

Adding second subtitle as original will break formatting and will not display the original text if text replica number (for example from .srt file) is not matching between the files.

Thanks!

**Comment by LeonCheung** [comment]
Like this?
![image](https://github.com/SubtitleEdit/subtitleedit/assets/494788/9d313036-ed24-4b0b-bdac-3b6881992977)

The second subtitle will be displayed on the video player.

**Comment by JustAndreww** [comment]
Almost, but I need to see the whole text of additional subtitles near the currently editing ones. Very similar to the translator mode, but without interaction.

Currently Video -> Open second subtitle file displays it only on video, but not in the main window with timings and texts.

Thanks!

**Reply by LeonCheung** [reply]
So, you are expecting a side-by-side view, and want to completely ignore the timecodes of the 2nd subtitle, in it, right?

**Reply by JustAndreww** [reply]
Sort of. I need the second (aka 'original') subtitles just be displayed as-is in the main view alongside with 'translate' subtitles, but in read-only mode. So when I'm adding line or removing it from 'translated' part nothing changes in 'original' part.

Hope that's understandable :)

**Reply by LeonCheung** [reply]
Try this:

Make sure the two subtitles have the same line count. Copy Then, go to menu > Tools > Make new empty translation from the current subtitle, then use the feature:

![image](https://github.com/SubtitleEdit/subtitleedit/assets/494788/b5e24172-ea1d-4d32-afb0-5f36f25a4345)

It still creates a translation, but it shares the same timecodes with the 1st subtitle (loaded already) nothing will break. You can decide whether to keep it or not.

And you may also want to toggle off this:

![image](https://github.com/SubtitleEdit/subtitleedit/assets/494788/0def994b-a5c7-4d5b-a75c-7e629ef135d4)

**Reply by JustAndreww** [reply]
Thanks for your help.

I can't make the original and translated subtitles with the same lines count. So your solution will not work, because in that case I would be navigating using the original sub timings which will not match the translated ones because of the lines shifting.

**Reply by LeonCheung** [reply]
Hmm, It makes no sense to do it all in SE if you can't make sure lines are perfectly matched.

In that case, you can first switch to layout No.2 or No.4 so that the text column is to the most right border of SE's UI, then open the 2nd subtitle in Notepad and place it next to SE. Problem solved?

**Reply by JustAndreww** [reply]
Yeah, like I'm doing right now :) The only missing thing is auto scroll. Would be nice to have this embedded with SE.

I'm not agree that there is no sense to have a different lines count and timings while doing the translation. Sometimes it's not possible to maintain the time slots exactly the same as original ones.

**Comment by MonsterSe7en** [comment]
I have a use-case here where I generate an AI transcript which can be quirky at times

Would be nice to refer it and create my own as a second proper subtitle

**Comment by MonsterSe7en** [comment]
https://github.com/SubtitleEdit/subtitleedit/issues/9793
